public/js/saveApplicant.js

```diff
@@ -46,11 +46,11 @@ let makeApiRequest = async (data) => {
 let validateFormInputs = (data) => {
     let validate = [];
 
-    validate.push(isName(data.name) && charsMaxLength(data.name));
+    validate.push(isName(data.name) && nameMaxLength(data.name));
     validate.push(isEmail(data.email));
     validate.push(isPhoneNumber(data.phoneNumber));
-    validate.push(isPresent(data.whyDev));
-    validate.push(isPresent(data.codeExperience));
+    validate.push(isPresent(data.whyDev) && textAreaMaxLength(data.whyDev));
+    validate.push(isPresent(data.codeExperience) && textAreaMaxLength(data.codeExperience));
 
```

Comment:
This is stupid haha, but the Mike and Charlie's notes bit should probably also be capped at 10k text haha.